atomics/T1048.003/T1048.003.yaml

```diff
@@ -67,6 +67,26 @@ atomic_tests:
       
           cat output_file | cut -d "A" -f 2 | cut -d " " -f 2 | cut -d "." -f 1 | sort | uniq | xxd -p -r
     name: manual
+- name: Exfiltration Over Alternative Protocol - DNS (Automated)
+  description: |
+    Exfiltration of specified file over DNS protocol.(Automated)
```

Comment:
Could you explain what the command does in more detail, like posts the data in chunks as the subdomain of a domain under the attackers control, resulting in numerous DNS requests to that domain. The attacker configures a server as an authorative name server for the domain and listens for DNS request, reconstruction the file from the many subdomain requests that it receives. (this is just me guessing, but something like that). Also, do you have a blog post you can link to in the description that will help users understand.